main-labs/sdn-openflow/pox/07-pox-host-discovery/controller/pox/ext/host_discovery.py

```diff
@@ -0,0 +1,134 @@
+import pox.openflow.libopenflow_01 as of
+from pox.core import core
+from pox.lib.recoco import Timer
+from pox.lib.addresses import EthAddr
+from pox.lib.packet.ethernet import ethernet
+from pox.lib.packet.arp import arp
+from pox.lib.addresses import IPAddr, EthAddr
+from pox.lib.util import dpidToStr
+from pox.lib.revent.revent import EventMixin
+from pox.lib.revent.revent import Event
+from datetime import datetime
+
+
+log = core.getLogger()
+MAX_HOST = 5
+MIN_SWITCH = 4
+
+class hostDiscovery:
```

Comment:
First letter of classes should be uppercase

Reply:
However, I noticed that POX uses this convention. So maybe is better to leave as you did.